File: BackEnd/Models/Cliente.cs
using System.Collections.Generic;

namespace LojinhaIT13.Models
{
    public class Cliente
    {
        public int ClienteId { get; set; }
        public string Nome { get; set; }
        public string Email { get; set; }
        public List<Pedido> Pedidos { get; set; }
        public Carrinho Carrinho { get; set; }
        public int CarrinhoId { get; set; }
    }
}

File: Front_end/lib/DTOs/cliente.dart
class Cliente {
  int? clienteId;
  String? nome;
  String? email;
  Cliente({this.clienteId, this.nome, this.email});
  Cliente.fromJson(Map<String, dynamic> json) {
    clienteId = json['clienteId'];
    nome = json['nome'];
    email = json['email'];
  }
  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = new Map<String, dynamic>();
    data['clienteId'] = this.clienteId;
    data['nome'] = this.nome;
    data['email'] = this.email;
    return data;
  }
}